components/builder-originsrv/src/migrations/origin_packages.rs

```diff
@@ -267,5 +267,35 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
+    migrator.migrate("originsrv",
```

Comment:
I may have missed some changes to the migration code but will this migration be picked up if its named `originsrv` which has already run?